src/scss/components/button/mixins/_hxButton.scss

```diff
@@ -186,13 +186,6 @@
   border: 0;
   color: $cyan-900;
 
-  // TODO: remove horizontal padding overrides so as to conform to redline specs
-  //
-  // Be aware that horizontal padding will affect the appearance of inline tertiary buttons.
-  // (e.g., <hx-file-tile> within a drop fence, "retry" button of <hx-file-tile>, etc.)
-  padding-left: 0;
-  padding-right: 0;
```

Comment:
@BY00565233, after thinking about this more please add these styles with a `deprecated` class.  Also, include a proper comment that they will be removed in `HelixUI-v1.0.0`.

Reply:
Changes have been done,Please review